10_files_lab/fileslab.txt

```diff
@@ -7,31 +7,66 @@ Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
   - human.h, monster.h
   - .highscore
 
-Write here the commands you used:
+  Write here the commands you used:
+
+  mkdir lab
+  cd lab
+  touch a.out
+  touch hero.c
+  touch enemy.c
+  touch game.c
+  touch main.c
+  touch human.h
+  touch monster.h
+  touch .highscore
+
 
 2. "cd" into lab folder and inside create the following subdirectories:
   - Music
   - Misc
   - Drivers
 
 Write here the commands you used:
+mkdir Music Misc Drivers
+ 
 
 3. Copy the file "/etc/passwd" to each subdirectory created in (2).
 
 Write here the commands you used:
+cd Music
+cp /etc/passwd .
+cd ..
+cd Misc
+cp /etc/passwd .
+cd ..
+cd Drivers
+cp /etc/passwd .
+cd ..
 
 4. Create a file named myinfo and inside write your name and 3 things you like about Unix.
 Now create a new directory named "myinfo" and move the file into it.
 
 Did it work? Why? Can you create a file "myinfo" inside a directory named "myinfo" ? How ?
 
+pico myinfo.txt
+mkdir myinfo
```

Comment:
note that there is a difference between "myinfo.txt" and "myinfo"..
in unix the extension is completely optional, as you saw with ```/etc/passwd```

the idea was to show that you can't have a file and a dir under the same path share the same name...